src/bin/tools/grant-exec-perms.ts

```diff
@@ -1,10 +1,11 @@
 import { getProjectRoot } from "./getProjectRoot";
 import { join as pathJoin } from "path";
-import * as child_process from "child_process";
-import * as fs from "fs";
+import { chmodSync, statSync, constants } from "fs";
 
-Object.entries<string>(JSON.parse(fs.readFileSync(pathJoin(getProjectRoot(), "package.json")).toString("utf8"))["bin"]).forEach(([, scriptPath]) =>
-    child_process.execSync(`chmod +x ${scriptPath}`, {
-        "cwd": getProjectRoot()
-    })
-);
+import(pathJoin(getProjectRoot(), "package.json")).then(({ bin }) => {
```

Comment:
you can import json files just like modules, no need to battle with fs.read or json.parse

Reply:
All right,  
it's just that it's more conveignent to read synchronously.  

But I don't mind you change ⬆️